usr/src/cmd/mdb/aarch64/mdb/mdb_aarch64util.c

```diff
@@ -300,50 +282,35 @@ mdb_aarch64_kvm_stack_iter(mdb_tgt_t *t, const mdb_tgt_gregset_t *gsp,
 		if ((argc != 0) &&
 		    ((args_style = saveargs_has_args(ins, insnsize, argc,
 		    start_index)) != SAVEARGS_NO_ARGS)) {
```

Comment:
`start_index` looks uninitialised now.

Reply:
Yup, I cleaned the libproc side up relying on the GCC warnings and didn't think to check they weren't disabled, on the mdb side.  We don't need this anymore.  Thanks